src/msg/simple/Pipe.cc

```diff
@@ -160,10 +160,7 @@ Pipe::Pipe(SimpleMessenger *r, int st, PipeConnection *con)
     connection_state->pipe = get();
   }
 
-  if (randomize_out_seq()) {
-    lsubdout(msgr->cct,ms,15) << "Pipe(): Could not get random bytes to set seq number for session reset; set seq number to " << out_seq << dendl;
```

Comment:
Also really the error handling we DID have was horrible. If we can't get actual random bytes in a cryptographic context it's better to just halt, or at least refuse the operation.